packages/toolkit/src/lib/vdp-sdk/helper/createInstillAxiosClient.ts

```diff
@@ -27,10 +27,10 @@ export function createInstillAxiosClient(
     );
   }
 
-  let APIVersion = env("NEXT_PUBLIC_GENERAL_API_VERSION");
+  let APIVersion = process.env.NEXT_PUBLIC_GENERAL_API_VERSION;
 
-  if (isModelEndpoint) {
-    APIVersion = env("NEXT_PUBLIC_MODEL_API_VERSION");
+  if (isAlphaVersion) {
+    APIVersion = process.env.NEXT_PUBLIC_MODEL_API_VERSION;
```

Comment:
Please reference how we set up a separate APIClient using the new sdk here https://github.com/instill-ai/console/blob/74ec0586da2025cbe0bf975f8c577ab164c0b0db/packages/toolkit/src/lib/vdp-sdk/index.ts#L40

I believe we also need to have a getInstillArtifactAPIClient here